sync_coda_to_engagement_db.py

```diff
@@ -43,4 +44,5 @@
     engagement_db = pipeline_config.engagement_database.init_engagement_db_client(google_cloud_credentials_file_path)
     coda = pipeline_config.coda_sync.coda.init_coda_client(google_cloud_credentials_file_path)
 
+    ensure_coda_datasets_exists(user, coda, coda_config)
```

Comment:
Do we want this to always run or put in a separate manually triggered tool?
I'm not sure what the best option is, happy to experiment with fully automatic for now if think that's best

Reply:
We'll experiment with fully automatic for now.